packages/fxa-settings/src/components/Nav/index.stories.tsx

```diff
@@ -4,12 +4,22 @@
 
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { MockedProvider } from '@apollo/client/testing';
+import { createCache } from '../../models/_mocks';
 import { Nav } from '.';
 
 storiesOf('Components|Nav', module)
   .add('basic', () => (
-    <Nav hasSubscription={false} primaryEmail="user@xample.com" />
+    <MockedProvider cache={createCache()}>
+      <Nav />
+    </MockedProvider>
```

Comment:
one downside about `useAccount` over props is the need to have a `MockedProvider` in tests and storybook. i wonder if/how we could reduce the boilerplate.

Reply:
At first glance I liked the idea of extending `MockedProvider` and passing in props to override the defaults we set, but I kind of feel like `cache={createCache({ // ... overrides` is more explicit and clear on what's going on. Not a great answer, I know... I'll put more thought into this. 🤔